app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,38 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+
+    sqlu"""
```

Comment:
Wasn't trying to imply there was something wrong - just wanted to know if there was a difference